src/openforms/forms/api/viewsets.py

```diff
@@ -388,6 +391,47 @@ def export(self, request, *args, **kwargs):
         response["Content-Length"] = len(response.content)
         return response
 
+    @extend_schema(
+        summary=_("Set default language for form"),
+        tags=["forms"],
+        request=None,
+        responses={status.HTTP_201_CREATED: FormSerializer},
```

Comment:
you'll need to use/define a proper serializer for the response data here, and use HTTP 200 instead of 201